src/Illuminate/Testing/Fluent/Concerns/Matching.php

```diff
@@ -106,8 +106,8 @@ public function whereAllType(array $bindings): self
     /**
      * Asserts that the property contains the expected values.
      *
-     * @param string       $key
-     * @param array|string $expected
+     * @param  string        $key
```

Comment:
```suggestion
     * @param  string  $key
```

Always standard two space